instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/CoreRerouteTest.kt

```diff
@@ -123,36 +140,53 @@ class CoreRerouteTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.jav
                         reasons: List<RouterFailure>,
                         routeOptions: RouteOptions
                     ) {
-                        // no impl
+                        logE(Tag("DEBUG"), Message("onFailure reasons=$reasons"))
```

Comment:
very interesting `Tag`s in the PR 😄 could you rename them to a specific

Reply:
This is a 🍒

And it's tests (we can fix in a separate PR if needed) 😅

Reply:
yeah, I know, but it's still an issue :)

Reply:
Could you cut a ticket? BTW Using `DEBUG` in tests is not a problem for me 😬